provider/resources.go

```diff
@@ -168,3 +172,22 @@ func Provider() tfbridge.ProviderInfo {
 
 //go:embed cmd/pulumi-resource-libvirt/bridge-metadata.json
 var metadata []byte
+
+var networkModesRegexp = regexp.MustCompile("- `[a-z]*`: ")
+
+func docRuleEdits(defaults []tfbridge.DocsEdit) []tfbridge.DocsEdit {
+	return append(defaults, tfbridge.DocsEdit{
+		Path: "network.markdown",
+		Edit: func(_ string, content []byte) ([]byte, error) {
+			matches := networkModesRegexp.FindAllString(string(content), -1)
+			for _, match := range matches {
+				after := strings.ReplaceAll(match, "`", "\"")
+				content = bytes.ReplaceAll(content,
+					[]byte(match),
+					[]byte(after),
+				)
+			}
+			return content, nil
```

Comment:
It feels like we could do this much more simply with https://pkg.go.dev/regexp#Regexp.ReplaceAllFunc:

```suggestion
			return networkModesRegexp.ReplaceAllFunc(content, 
				func(src []byte) []byte {
					return bytes.ReplaceAll(src, 
						[]byte("`"), []byte(`"`))
			}), nil
```

It's possible I'm missing some reason why this isn't possible, but it looks very possible to me.